modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -174,7 +174,7 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:) # rubocop:disable
             claim_type_code: data[:bnft_claim_type_cd],
             claim_type: data[:claim_status_type],
             close_date: data[:claim_complete_dt].present? ? format_bgs_date(data[:claim_complete_dt]) : nil,
-            contention_list: data[:contentions]&.split(','),
+            contention_list: data[:contentions].present? ? data[:contentions].split(',').map!(&:strip) : nil,
```

Comment:
let's make this
`data[:contentions]&.split(',')&.collect(&:strip) || []`
VA Gov asked that this not be possible to be nil so we'll want an empty array if nil. And I believe our documentation does not reflect that either. Once those changes are done, we'll need to run the OAS RSWAG updates. This should be an array of items, not an array of one string comma separated. 

Below it shows: 
```
["abnormal heart (New), abscess kidney (New), encephalitis lethargica residuals (New), dracunculiasis (New), gingivitis (New), abnormal weight loss (New), groin condition (New), metritis (New)"]
```
it should be
```
  "abnormal heart (New)",
  "abscess kidney (New)",
  "encephalitis lethargica residuals (New)",
  .......
]
```